lib/models/command.js

```diff
@@ -237,9 +239,15 @@ let Command = CoreObject.extend({
 
     this._currentTask = task;
 
-    return Promise.resolve(task.run(options)).finally(() => {
-      delete this._currentTask;
-    });
+    return Promise.resolve(task.run(options))
+      .catch(error => {
+        logger.info(`An error occurred running \`${name}\` from the \`${this.name}\` command.`, error.stack);
```

Comment:
Ya. Those only show the original stack of the error. While spelunking a fairly annoying bug with @tomdale we had a situation where the stack was completely in native code land and nothing pointed to the actual command / task that was causing the error. 

In our case, this was due to native promises suffering from similar issues as we had with RSVP waaaay back (swallowed stacks and what not). It's likely that this will be corrected, but in the meantime I figured we could just log the error stack and mention which command / task triggered it.